index.js

```diff
@@ -1,7 +1,26 @@
 module.exports = editorConfigToPrettier;
 
+function removeUnset(editorConfig) {
+  const result = {};
+  const keys = Object.keys(editorConfig);
+  for (let i = 0; i < keys.length; i++) {
+    const key = keys[i];
+    if (editorConfig[key] === "unset") {
+      continue;
+    }
+    result[key] = editorConfig[key];
+  }
+  return result;
+}
```

Comment:
We could probably use something like https://www.npmjs.com/package/omit instead of this, but it's ok for now.

Reply:
Sure, I just don't have the habit of adding dependencies to my projects

Let me know what you prefer